config/config.default.php

```diff
@@ -162,7 +163,7 @@
  * You will need to change some settings inside .htaccess
  */
 
-$settings["pretty_url"] = false;
```

Comment:
Since this requires additional changes, it should maybe stay at false for default.